packages/cli-kit/src/public/node/output.ts

```diff
@@ -251,10 +279,11 @@ export const completed = (content: Message, logger: Logger = consoleLog) => {
  * Ouputs debug information to the user. By default these output is hidden unless the user calls the CLI with --verbose.
  * Debug messages don't get additional formatting.
  * Note: Debug messages are sent through the standard output.
+ *
  * @param content - The content to be output to the user.
  * @param logger - The logging function to use to output to the user.
  */
-export const debug = (content: Message, logger: Logger = consoleLog) => {
+export function debug(content: Message, logger: Logger = consoleLog): void {
```

Comment:
100% agree. We should refrain from doing `import * as output from "..."` and import the elements that are used.

Reply:
I had the same doubt! I finally decided to leave the original names for two reasons:
- As you said, most of the times we import `* as output` because we usually require several functions (debug, info, token, content...), so `output.success` is clear.
- I think most of the functions are also easy to understand as they are: debug, info, warn, token, content, formatPackageManagerCommand, stringifyMessage... I agree success could be confusing, though.

So I don't know, do you prefer to rename them? Or maybe only some of them? I don't have a strong opinion.

Reply:
After checking all the imports, most of them just use a few functions (max 5 or 6), so we are fine 🙂